src/size-common.c

```diff
@@ -40,7 +40,7 @@ static
 SEXP vctrs_size2_common(SEXP x, SEXP y, struct counters* counters, void* data);
 
 r_ssize vec_size_common(r_obj* xs, r_ssize absent) {
-  r_obj* common = KEEP(reduce(R_NilValue, args_empty, xs, &vctrs_size2_common, NULL));
+  r_obj* common = KEEP(reduce(R_NilValue, args_empty, NULL, xs, &vctrs_size2_common, NULL));
```

Comment:
And here too in `vec_size_common()`. I feel like this one would make sense. Maybe `vec_recycle_common()` too since I often do things like `args <- vec_cast_common(!!!args); args <- vec_recycle_common(!!!args)` back to back and it would be nice to be able to pass an arg name through both

Reply:
yeah `vec_size_common()` and `vec_recycle_common()` need custom error contexts as well, I'll do that.